tests/src/Interop/IJW/ManagedCallingNative/ManagedCallingNative.csproj

```diff
@@ -24,6 +24,9 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
   </PropertyGroup>
+  <PropertyGroup>
+    <CopyDebugRuntimeDllsToOutputDirectory Condition="('$(Configuration)' == 'Debug') Or ('$(Configuration)' == 'Checked')">true</CopyDebugRuntimeDllsToOutputDirectory>
```

Comment:
`CopyDebugRuntimeDllsToOutputDirectory` - this name is ambiguous. I would prefer something like `CopyDebugCRTDllsToOutputDirectory`. Basically the name "Runtime" implies the opposite of what people originally assume.